wicket-jquery-ui-samples/src/main/java/com/googlecode/wicket/jquery/ui/samples/pages/kendo/menu/KendoContextMenuPage.html

```diff
@@ -13,6 +13,12 @@
 	<div wicket:id="menu"></div>
 	<br/>
 	<div wicket:id="feedback"></div>
+
+	<wicket:fragment wicket:id="moveToPosition">
+		<form wicket:id="form" onclick="var $evt=jQuery.event.fix(event);if($evt.target.nodeName !== 'BUTTON'){$evt.stopPropagation()}">
```

Comment:
But yes, we can provide such form and users can extend from it. If someone needs more sophisticated solution then (s)he can use his/her own Form.